sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py

```diff
@@ -212,22 +212,35 @@ def allow_replanning(self, value):
     def execute(self):
         """
         Executes the last plan made.
+        @return - Success of execution.
         """
         if self.check_plan_is_valid():
-            self._move_group_commander.execute(self.__plan)
+            success = self._move_group_commander.execute(self.__plan)
             self.__plan = None
+            if not success:
+                rospy.logerr("Execution failed.")
+                return success
+            rospy.loginfo("Execution succeeded.")
+            return success
```

Comment:
The block could be simplified a bit:

if not success:
    rospy.logerr()
else:
    rospy.loginfo()
return success